src/SiteInformation.php

```diff
@@ -60,4 +60,24 @@ public function getSiteOwner(): ConceptInterface {
     return $this->entityStorage->load($id);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function hasContentOwners(): bool {
```

Comment:
According to my understanding, this configuration serves as a default value definition. I would reflect this in the name of the function like: hasDefaultContentOwners().